app/presenters/organisations/show_presenter.rb

```diff
@@ -0,0 +1,59 @@
+module Organisations
+  class ShowPresenter
```

Comment:
There is a lot of logic in this presenter, I think it's worth writing some tests for.

Reply:
Added. It'll need expanding and modifying over time, but it's started.